docs/content/author-bundles.md

```diff
@@ -316,6 +318,26 @@ image: "{{bundle.images.websvc.repository}}@{{bundle.images.websvc.digest}}"
 
 At runtime, these will be updated appropriately if a bundle has been [copied](/copy-bundles). Note that while `tag` is available, you should prefer the use of `digest`.
 
+Here is a breakdown of all the supported fields on an image in this section of the manifest:
+
+* `description`: a description of the image
+* `imageType`: the type of image (most commonly, `"docker"`)
```

Comment:
I recommend removing the code blocks from the description text: `docker`, `REGISTRY/ORG/IMAGE/` and `digest`.

It makes it hard to read (see screenshot). In general, I recommend using code formatting for text that we want users to copy/paste to run themselves, or for something that needs to be formatted in monospace like console output or actual code blocks. Otherwise highlighting random words in a paragraph as code is distracting and the formatting doesn't usually help improve reading comprehension.

At a later point, I would like to cleanup how we format our doc so that for example we don't code format the names of fields in the bulleted lists either. But for now it's consistent.

![Screen Shot 2020-07-17 at 11 34 10 AM](https://user-images.githubusercontent.com/1368985/87809939-d1f21280-c821-11ea-8301-0ac2da7501d8.png)

Reply:
Updated!